index.js

```diff
@@ -180,7 +182,7 @@ RadixRouter.prototype = {
 
       if (nextNode !== undefined) {
         node = nextNode
-      } else if (i === sections.length - 1) {
+      } else if (i === endSections) {
```

Comment:
`sections - 1` getting executed for every loop iteration where `nextNode === 'undefined'`